code-challenges.test.js

```diff
@@ -1,67 +1,326 @@
+// const { it } = require("node:test")
+// const { describe } = require("yargs")
+ 
 // Please read all questions thoroughly
 // Pseudo coding is REQUIRED
 // If you get stuck, please leave comments to help us understand your thought process
-
+ 
 // Use test driven development to complete the following questions
 // Add appropriate dependencies: $ yarn add jest
-
+ 
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
-
+ 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
-
+ 
 // a) Create a test with expect statements for each of the variables provided.
-
+ 
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
-
+ 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
-
+ 
+// RED
+ // Write the test
+   describe("fibFinder", () => {
+     it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+       const fibLength1 = 6
+       // Expected output: [1, 1, 2, 3, 5, 8]
+       const fibLength2 = 10
+       // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+       expect(fibFinder(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+       expect(fibFinder(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+     })
+   })
+ 
+   // See the test fail
+     // FAIL  ./code-challenges.test.js
+     // fibFinder
+     //   ✕ takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence
+ 
+     // ● fibFinder › takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence
+    
+     //   ReferenceError: fibFinder is not defined
+  
 // b) Create the function that makes the test pass.
+ 
+ // GREEN
+   //PSEUDOCODE:
+     // declare a function called fibFinder
+     // input: a number
+     // output: return a an array with the length of elements the the number in the input is asking for and give the Fibonacci sequence
+     // PROCESS: 
+     // Create a function called "fibFinder"
+     // declare a variable called "array" and assign the value of and array [1 , 1], this allowed me to get started and have an array in place to work on. In the beginning I tryed to declare variable with the beginning numbers of the Fibonacci sequence.
+     // I used the for loop method to iterate through the array through each element.
+     // after in order to get the elements to add together I took the element at the zeroth index and then added the the following index creating a new element. In order to add this new element into the array I used the push method in order to add to the current array, then called on the array that was updated. With taking the the given arrays in they tell the funtion which element to stop at. 
```

Comment:
Your pseudocode throughout this whole assessment is beautiful! If I am able to take the journey with you in writing this code, than I am sure it was a breeze to write the code after! Great job!